typedjsonrpc/registry.py

```diff
@@ -37,10 +37,10 @@ class Registry(object):
     :type tracebacks: dict[int, werkzeug.debug.tbtools.Traceback]
     """
 
-    json_encoder = json.JSONEncoder
+    json_encoder = json.JSONEncoder()
     """The JSON encoder class to use.  Defaults to :class:`json.JSONEncoder`"""
```

Comment:
Change docs to remove the word class since it's an instance of an JSONEncoder now.